/**:
  ros__parameters:
    tag_family: 'tag36h11'
    apriltag_ids: ['tag36h11:1']
    destination_ids: ['kitchen', 'table1', 'table2', 'table3']

    tag36h11:1_translation: [5.0, 5.0, 0.0]
    tag36h11:1_rotation: [0.0, 0.0, 0.0]
    
    kitchen_coords: [0.0, 0.0, 0.0]
    table1_coords: [2.0, 3.0, 0.0]
    table2_coords: [0.0, -2.0, 0.0]
    table3_coords: [4.0, 1.0, 0.0]